R/bs_sass.R

```diff
@@ -57,53 +57,38 @@ bootstrap <- function(theme = bs_theme_get(),
 
   theme <- as_bs_theme(theme)
   version <- theme_version(theme)
-  minified <- isTRUE(sass_options$output_style %in% c("compressed", "compact"))
-  output_css <- if (minified) "bootstrap-custom.min.css" else "bootstrap-custom.css"
-  js <- bootstrap_javascript(version, minified)
 
-  cache_key <- sass::sass_hash(list(
-    theme,
-    sass_options,
-    get_exact_version(version),
-    utils::packageVersion("bootstraplib")
-  ))
-
-  # Temp dir for building the HTML dependencies
-  output_path <- file.path(
-    tempdir(),
-    "bootstraplib",
-    paste0("bootstrap-", cache_key)
-  )
-
-  # If the output path already exists, then we don't need to write anything.
-  if (!dir.exists(output_path)) {
-    dir.create(output_path, recursive = TRUE)
-
-    # Compile sass in temp dir
-    sass::sass(
-      input = theme,
-      options = sass_options,
-      output = file.path(output_path, output_css),
-      cache = cache,
-      write_attachments = TRUE
+  out_file <- sass::sass(
```

Comment:
The logic has changed here so that it always overwrites the files in the bootstraplib-xxx directory, right? It would be good to check if there's a noticeable performance penalty.